tox.ini

```diff
@@ -14,7 +14,7 @@ addopts =
     --cov pyls --cov test
 
 [tox]
-envlist = py27,py35,py36
+envlist = py27,py36
```

Comment:
Would it not be better to test against 3.4 ? The test suite passes here, and 3.6 is not yet widely available.